portal/src/main/webapp/js/src/patient-view/util/util.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+/**
+ * Get first key found in object. Otherwise return null.
+ * @param {object} object - object with key/value pairs
+ * @param {array} keys - array of keys
+ */
+window.getFirstKeyFound = function(object, keys) {
```

Comment:
I would suggest to add all functions into patientViewUtil instead of registering into global.If you think other pages can benefit from this function, put it under cbio-util.js